src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -152,7 +153,17 @@ const BrazeBannerWithSatisfiedDependencies = ({
     meta,
 }: InnerProps) => {
     useEffect(() => {
+        // Log the impression with Braze
         meta.logImpression();
+
+        // Log VIEW event with Ophan
+        submitComponentEvent({
+            component: {
+                componentType: 'BRAZE_BANNER',
+                id: meta.dataFromBraze.componentName,
```

Comment:
For now this will only ever be one value: `DigitalSubscriberAppBanner` since that's the only component we currently support.